pages/learn/manage/configuration.md

```diff
@@ -22,6 +22,7 @@ RESIN_SUPERVISOR_LOG_CONTROL | true | Enable / Disable logs being sent to the {{
 RESIN_SUPERVISOR_POLL_INTERVAL | 900000 | Define the {{ $names.company.lower }} API poll interval in milliseconds. This interval will only matter if the device is not connected to the VPN at the time an update is pushed, or if RESIN_SUPERVISOR_INSTANT_UPDATE_TRIGGER is set to false. Starting from supervisor v9.13.0, the supervisor will use a random time between 0.5 and 1.5 times this poll interval each time it checks the balenaCloud API. The minimum value for this variable is defined by the balenaCloud backend, and may vary. | v1.3.0
 RESIN_SUPERVISOR_VPN_CONTROL | true | Enable / Disable VPN | v1.3.0
 RESIN_SUPERVISOR_INSTANT_UPDATE_TRIGGER | true | Enable / Disable instant triggering of updates when a new release is deployed. If set to false, the device will ignore the notification that is triggered when the device's target state has changed. In this case, the device will rely on polling to apply updates. Coupled with a large RESIN_SUPERVISOR_POLL_INTERVAL, this allows spreading out updates in large fleets to avoid overloading local networks when there is a large number of devices at one location. | v9.13.0
+RESIN_OVERRIDE_LOCK | 0 | When set to 1 overrides any existing update lock on the device. Allows updating devices in the case that the application locked updates but is stuck in an invalid state. |
```

Comment:
Should we specify RESIN_SUPERVISOR_OVERRIDE_LOCK as well, since both are valid?

Reply:
@cywang117 that's not yet whitelisted in the API's definitions though :thinking: 
![image](https://user-images.githubusercontent.com/1295829/121324276-3f66d000-c919-11eb-9186-cbdf9269bff3.png)

Atm I would prefer to keep it consistent with what the deprecated custom endpoint was doing, so that it will be easier to migrate all user configs to the new RESIN_SUPERVISOR_OVERRIDE_LOCK config var at a later point.